### ISSUE TITLE
initial trust on first use

### ISSUE THREAD

**Issue by florianeichin** [notaryproject/notary#1294]
Wouldn't it make sense to ask the user while first pull if the used public part of the rootkey should be trusted, similar to first ssh usage? 
As most of the security magic happens in background thourgh notary, the normal user has no possibility to verify the root of trust when first pulling an image. With a cmd question the attention for a secure image would grow, maybee.

**Comment by HuKeping** [comment]
The [trusted pin](https://github.com/theupdateframework/notary/blob/9763f6f2724e01e6310098ab0031964e30346e14/docs/reference/client-config.md#trust_pinning-section-optional) might be help?

**Comment by florianeichin** [comment]
yeah, I know notary is using TOFU for initial trust, but isn't it (in worst case) kind of blind trust, if the user doesn't approve, that the used public key is the one he want's to trust?

**Comment by justincormack** [comment]
You don't have to use TOFU, you can trust pin via CA or a list of certificates, see https://github.com/theupdateframework/notary/blob/master/trustpinning/trustpin.go#L13-L34
